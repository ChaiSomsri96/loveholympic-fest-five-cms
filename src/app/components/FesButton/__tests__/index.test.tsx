import * as React from 'react';
import { render } from '@testing-library/react';

import { FesButton } from '..';

jest.mock('react-i18next', () => ({
  useTranslation: () => {
    return {
      t: str => str,
      i18n: {
        changeLanguage: () => new Promise(() => {}),
      },
    };
  },
}));

describe('<FesButton  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(
      <FesButton style={{}} title="" onClick={() => {}} icon={<></>} />,
    );
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
