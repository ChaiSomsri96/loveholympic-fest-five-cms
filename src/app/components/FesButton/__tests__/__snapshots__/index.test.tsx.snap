// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FesButton  /> should match snapshot 1`] = `
.c0 {
  font-style: normal;
  font-weight: bold;
  font-size: 16px;
  line-height: 19px;
  border: none;
  border-radius: 6px;
  margin: 0px 0px 0px 15px;
  padding: 10px 15px;
  background: #2A4365;
}

<button
  class="c0 btn btn-secondary"
  type="button"
>
   
  
</button>
`;
